rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -1222,6 +1219,24 @@ public JobId copyJobAndResubmitWithGeneralInfo(JobId jobId, Map<String, String>
         return null;
     }
 
+    @Override
+    public Job getJobContent(JobId jobId) throws NotConnectedException, PermissionException, UnknownJobException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)